### DISCUSSION TITLE
"Current Demand" to be used for solar.

### DISCUSSION THREAD

**Discussion by joel-graham** [BottlecapDave/HomeAssistant-OctopusEnergy#928] [Q&A]
My current demand will tell me my negative consumption when producing solar, whilst it's not 100% accurate, it's certainly better than _not_ having this data which is my current state. Is there a way to use this data, when negative, in the energy dashboard. Currently there's no way, that I can see, to use this in the "Energy" dashboard.

I'm very new to HA, so please forgive any ignorance on my part.

**Comment by BottlecapDave** [comment]
Hi sorry for the very late response. The demand data is available in watts which is not compatible with the energy dashboard. You could potentially use a utility meter to aggregate this data into a Wh or kWh format which could then be added to the energy dashboard. This would just be an estimate due to the frequency that the demand sensor updates.

**Reply by joel-graham** [reply]
Thank you for your response; no need to apologise!

I'll have a look into your suggestion, aware it's only an estimate.
